site/src/components/pages/docs/search/index.tsx

```diff
@@ -143,32 +153,30 @@ export const Search: React.VoidFunctionComponent<SearchProps> = ({
     ) : undefined;
 
   return (
-    <>
-      <TextField
-        inputRef={inputRef}
-        onKeyDown={(event: KeyboardEvent<HTMLInputElement>) => {
-          if (searchResults.length > 0) {
-            if (event.key === "ArrowUp") {
-              event.preventDefault();
-
-              if (activeResult === 0) {
-                return setActiveResult(searchResults.length - 1);
-              }
-
-              setActiveResult(activeResult - 1);
-            }
-
-            if (event.key === "ArrowDown") {
-              event.preventDefault();
-
-              if (activeResult > searchResults.length - 1) {
-                return setActiveResult(0);
-              }
+    <Box
+      onKeyDown={(event: KeyboardEvent<HTMLInputElement>) => {
```

Comment:
I've also spotted a bug where the up button doesn't immediately work after I've entered a search result. Can you reproduce this?

Reply:
Yes, thank you. Addressed in 6c5dcd2298c638618c1b62fb2e49950aee5b1640